examples/call_raw/test/tests.ts

```diff
@@ -15,12 +15,7 @@ export function getTests(call_raw_canister: ActorSubclass<_SERVICE>): Test {
                 0n
             );
 
-            if ('Err' in result) {
-                throw new Error(result.Err);
-            }
-
-            expect(result).toHaveProperty('Ok');
-            expect(result.Ok).toMatch('blob');
+            expect(result).toMatch('blob');
```

Comment:
What kind of errors might we get from call raw? Should we add some tests to make sure those error throw correctly?

Reply:
Issue created to address this more broadly: https://github.com/demergent-labs/azle/issues/1863